docs/ops/detection/ROIAlign_9.md

```diff
@@ -0,0 +1,117 @@
+# ROIAlign {#openvino_docs_ops_detection_ROIAlign_9}
+
+**Versioned name**: *ROIAlign-9*
+
+**Category**: *Object detection*
+
+**Short description**: *ROIAlign* is a *pooling layer* used over feature maps of non-uniform input sizes and outputs a feature map of a fixed size.
+
+**Detailed description**: [Reference](https://arxiv.org/abs/1703.06870).
+
+*ROIAlign* performs the following for each Region of Interest (ROI) for each input feature map:
+1. Multiply box coordinates with *spatial_scale* to produce box coordinates relative to the input feature map size based on *aligned_mode* attribute.
+2. Divide the box into bins according to the *sampling_ratio* attribute.
+3. Apply bilinear interpolation with 4 points in each bin and apply maximum or average pooling based on *mode* attribute to produce output feature map element.
+
+**Attributes**
+
+* *pooled_h*
+
+  * **Description**: *pooled_h* is the height of the ROI output feature map.
+  * **Range of values**: a positive integer
+  * **Type**: `int`
+  * **Required**: *yes*
+
+* *pooled_w*
+
+  * **Description**: *pooled_w* is the width of the ROI output feature map.
+  * **Range of values**: a positive integer
+  * **Type**: `int`
+  * **Required**: *yes*
+
+* *sampling_ratio*
+
+  * **Description**: *sampling_ratio* is the number of bins over height and width to use to calculate each output feature map element. If the value
+  is equal to 0 then use adaptive number of elements over height and width: `ceil(roi_height / pooled_h)` and `ceil(roi_width / pooled_w)` respectively.
+  * **Range of values**: a non-negative integer
+  * **Type**: `int`
+  * **Required**: *yes*
+
+* *spatial_scale*
+
+  * **Description**: *spatial_scale* is a multiplicative spatial scale factor to translate ROI coordinates from their input spatial scale to the scale used when pooling.
+  * **Range of values**: a positive floating-point number
+  * **Type**: `float`
+  * **Required**: *yes*
+
+* *mode*
+
+  * **Description**: *mode* specifies a method to perform pooling to produce output feature map elements.
+  * **Range of values**:
+    * *max* - maximum pooling
+    * *avg* - average pooling
+  * **Type**: string
+  * **Required**: *yes*
+
+  * *aligned_mode*
+
+  * **Description**: *aligned_mode* specifies how to transform the coordinate in original tensor to the resized tensor.
+  * **Range of values**: name of the transformation mode in string format (here scale[x] is resized_shape[x] / original_shape[x] and x_original is a coordinate in axis x, for any axis x from the input axes):
```

Comment:
could you please elaborate more about `resized_shape`?

Reply:
have added more description for this